lahja/asyncio/endpoint.py

```diff
@@ -544,29 +538,42 @@ def _process_item(self, item: BaseEvent, config: Optional[BroadcastConfig]) -> N
 
         if event_type in self._queues:
             for queue in self._queues[event_type]:
+                self.logger.info("PUT IN QUEUE %s", queue)
                 queue.put_nowait(item)
 
         if event_type in self._handler:
             for handler in self._handler[event_type]:
                 handler(item)
 
+    def stop_server(self) -> None:
```

Comment:
Feels like this API should be private to discourage using it over the `Endpoint.run` API.